app/assets/javascripts/scp-related-genes-ideogram.js

```diff
@@ -13,8 +13,21 @@
 
 /** Handle clicks on Ideogram annotations */
 function onClickAnnot(annot) {
+  const ideo = this // eslint-disable-line
   document.querySelector('#search_genes').value = annot.name
-  document.querySelector('#perform-gene-search').click()
+
+  // Enable merge of related-genes log props into search log props
+  // This helps profile the numerator of click-through-rate
+  const namespacedProps = {}
+  const props = getRelatedGenesAnalytics(ideo)
+  Object.entries(props).forEach(([key, value]) => {
+    namespacedProps[`relatedGenes:${key}`] = value
+  })
+
+  const event = Object.assign(
+    { type: 'click-related-genes' }, namespacedProps
+  )
```

Comment:
```suggestion
 namespacedProps['type'] = 'click-related-genes'
```
namespacedProps is a fresh object and you're not using it anywhere else, so no need for the extra copy